### PR TITLE
feat: add application stack variable

### PR DESCRIPTION
**Pull request by stiross79**
Added the possibility to define an application stack to define a docker container to be used by the app.

### Checklist

- [x] I've updated both the `azurerm_linux_web_app` and `azurerm_windows_web_app` resources.

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -35,6 +35,18 @@ variable "app_settings" {
   default     = null
 }
 
+variable "application_stack" {
+  description = "A map of application stack settings for this Web App."
```
Comment (hknutsen) [inline variables.tf:39]:
```suggestion
  description = "An object of application stack settings for this Web App. Note that application stack settings are often configured outside of Terraform (for example when deploying code), so configuring application stack settings in Terraform may cause conflicts."
```

**Review by hknutsen — CHANGES_REQUESTED** [review]
Implementation looks perfect 🎉 However, I've suggested an addition to the variable description to clarify any confusion around potential conflicts.

**Comment by helenakallekleiv** [comment]
I've also run the tests, and everything is looking nice 🎉

**Comment by hknutsen** [comment]
Alternative solution: configure these values as app settings through #164

**Review by hknutsen — APPROVED** [review]
Looks good 👍

**Review by helenakallekleiv — APPROVED** [review]
Looking good 🌸

**Pull request merged**